docs/ref/settings.txt

```diff
@@ -323,7 +323,7 @@ protection is safe from cross-subdomain attacks by default - please see the
 Default: ``False``
 
 Whether to use ``HttpOnly`` flag on the CSRF cookie. If this is set to
-``True``, client-side JavaScript will not to be able to access the CSRF cookie.
+``True``, client-side JavaScript will not be able to access the CSRF cookie.
```

Comment:
We have the same typo in `SESSION_COOKIE_HTTPONLY` and `LANGUAGE_COOKIE_HTTPONLY`. Can you fix all of them?

Reply:
Done.